modules/service/import-service.js

```diff
@@ -640,6 +642,7 @@ class ImportService {
             otObject['@type'] = constants.objectType.otConnector;
         }
 
+        // todo add otJsonService
```

Comment:
comment

Reply:
This is a placeholder for new sorting method which should be implemented in the next sprint